backend/peakperformance_backend/src/main/java/com/peakperformace/peakperformance_backend/exercisesession/ExerciseSession.java

```diff
@@ -0,0 +1,89 @@
+package com.peakperformace.peakperformance_backend.exercisesession;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+import com.peakperformace.peakperformance_backend.exercise.Exercise;
+import com.peakperformace.peakperformance_backend.exercise.model.WeightReps;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.SequenceGenerator;
+
+@Entity
+@Table
+
+public class ExerciseSession {
+    @Id
+    @SequenceGenerator (
+        name = "exercisesession_sequence",
+        sequenceName = "exercisesession_sequence",
+        allocationSize = 1
+    )
+    @GeneratedValue (
+        strategy = GenerationType.SEQUENCE,
+        generator = "exercisesession_sequence"
+    )
+
+    private LocalTime timeOfExercise;
+    private LocalDate dateOfExercise;
```

Comment:
@CreationTimeStamp <-- put this annotation above the declaration
use LocalDateTime type instead, has both of these